client/sources/common/interpreter.py

```diff
@@ -320,6 +320,7 @@ def _compare(self, expected, code):
             output = printed.splitlines()
             actual = CodeAnswer(output=output)
 
+        self.cases_total += 1
```

Comment:
does moving this out here have side effects?

Reply:
cases_total was introduced for fpp, so it shouldn't have unexpected side effects.